package.json

```diff
@@ -12,12 +12,13 @@
   ],
   "scripts": {
     "start": "tsdx watch",
-    "build": "tsdx build",
+    "build": "yarn apollo-codegen && tsdx build",
```

Comment:
Types should be generally generated once. While developing i tend to run build many times and this adds additional overhead.

Reply:
yeah I was aware that was a problem but yeah having it separately is ok now there is a comment in the readme